python/ray/actor.py

```diff
@@ -29,6 +29,11 @@
     PythonFunctionDescriptor,
     raise_sys_exit_with_custom_error_message,
 )
+from ray.dag.class_node import (
+    PARENT_CLASS_NODE_KEY,
+    PREV_CLASS_METHOD_CALL_KEY,
+    ClassMethodNode,
+)
```

Comment:
For the test failure, I'm wondering if this is causing an increase in startup time?

Reply:
Hmm this just adds 1ms overhead when ray.init() is called. but let me try. nothing to lose...

Reply:
yeah this fixes the issue... I think it is really a bug from that test, and you should fix it, but I will just move import here for now